Source/Scene/Primitive.js

```diff
@@ -461,11 +461,18 @@ define([
     });
 
     function cloneAttribute(attribute) {
+        var clonedValues;
+        if(isArray(attribute.values)) {
+            clonedValues = attribute.values.slice(0);
+        }
+        else {
```

Comment:
`CTRL-ALT-L`.  Soon enough we will be implementing either `jscs` or `eslint` soon and then it will find and fix these errors automatically :)